src/routes/+page.svelte

```diff
@@ -1,2 +1,2 @@
-<h1>Welcome to SvelteKit</h1>
+<h1>Welcome to fortkörning</h1>
 <p>Visit <a href="https://kit.svelte.dev">kit.svelte.dev</a> to read the documentation</p>
```

Comment:
Remove the link to the sveltekit documentation.

Reply:
har fixat det